sa/model.go

```diff
@@ -41,7 +41,9 @@ type regModel struct {
 
 // challModel is the description of a core.Challenge in the database
 //
-// The Validation field is a stub; the column is only there for backward compatibility.
+// XXX(#983): The TLS field is a stub; the column is only there for backward
```

Comment:
No XXX per [dev guidelines](https://github.com/letsencrypt/boulder/wiki/Boulder-Development).

TODO if you meant to keep it up here instead of it being removed before you posted this PR.